### ISSUE TITLE
Syncplay doesn't run with mpv-git on Windows 10

### ISSUE THREAD

**Issue by da-moo** [Syncplay/syncplay#286]
I'm using scoop to install/update mpv-git (weekly builds by shinchiro, page here: https://sourceforge.net/projects/mpv-player-windows/), and I'm getting an error in the latest SyncPlay Client (1.6.4a) that my copy of mpv isn't compatible with SyncPlay. It's worked in the past, so I'm guessing something changed in the latest commits of mpv that broke something with Syncplay?

![image](https://user-images.githubusercontent.com/690524/78462869-6639ce80-76a4-11ea-87cb-20eace59b775.png)

`mpv --version`:

```
mpv 0.32.0-320-gbca917f6d2 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on Sun Mar 29 16:28:52 +08 2020
FFmpeg library versions:
   libavutil       56.42.102
   libavcodec      58.77.101
   libavformat     58.42.100
   libswscale      5.6.101
   libavfilter     7.77.101
   libswresample   3.6.100
FFmpeg version: git-2020-03-29-4cb0dda55
```

**Comment by Et0h** [comment]
Yes, see https://github.com/Syncplay/syncplay/issues/261 for details. For now, please use mpv 0.32.0 as per Syncplay homepage.

**Comment by ALi3naTEd0** [comment]
I'm having the same problem on Windows 10, seems like that version you're mentioning @Et0h is for linux... any suggestions for windows?

**Comment by blaenk** [comment]
Was this closed because it was fixed? I subscribed to it to know when I could go back to git mpv.

**Comment by Et0h** [comment]
It was closed because it is s duplicate of #261 - nobody has volunteered to fix it yet

**Comment by blaenk** [comment]
Thanks, I'll subscribe to that one 👍
